src/Renderer/PhpRenderer.php

```diff
@@ -69,7 +69,7 @@
  * @method string htmlPage($data, array $attribs = array(), array $params = array(), $content = null)
  * @method mixed|null identity()
  * @method \Laminas\View\Helper\InlineScript inlineScript($mode = \Laminas\View\Helper\HeadScript::FILE, $spec = null, $placement = 'APPEND', array $attrs = array(), $type = 'text/javascript')
- * @method \Laminas\View\Helper\Layout layout($template = null)
+ *  @method Model|\Laminas\View\Helper\Layout layout($template = null)
```

Comment:
I just want to note that the whitespace change at the front was not part of my PR so there is a slight diff here now

Reply:
My bad - didn't spot that when merging conflicts